tiledb/sm/serialization/query.cc

```diff
@@ -74,7 +74,7 @@
 #include "tiledb/sm/serialization/config.h"
 #include "tiledb/sm/serialization/fragment_metadata.h"
 #include "tiledb/sm/serialization/query.h"
-#include "tiledb/sm/storage_manager/storage_manager_declaration.h"
+#include "tiledb/sm/storage_manager/storage_manager.h"
```

Comment:
Does this not mess with the storage manager override when building unit tests?

Reply:
I don't think so. `storage_manager.h` is the file with the forwarding logic to the custom storage manager, and it was previously already included through `fragment_metadata.h`.

Reply:
Ah, right. Good call.